docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,42 +1,60 @@
 ### Workflow
 
-#### Contributions
+#### Gatekeepers
 
-All code changes and additions to Mbed OS are handled through GitHub. If you want to contribute, either by adding features or by fixing bugs, please follow the guidelines for [new features](#contributing-new-features-to-mbed-os) and [bugs](#reporting-and-fixing-bugs). In both cases, please follow the [code style guide and GitHub pull request guidelines](/docs/v5.7/reference/guidelines.html#style)</a>. Please also read the [CLA](/docs/v5.7/reference/guidelines.html#cla) guidelines because we will immediately close pull requests submitted without a CLA.
+##### Mbed OS maintainers
 
-#### Contributing new features to Mbed OS
+The maintainers are a small group of Mbed OS engineers who are responsible for the Mbed OS codebase. Their primary role is to progress contributions, both internal and external, from the initial pull request state through to released code. 
 
-Before contributing an enhancement (new feature, new port and so on), please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on a related feature.
+Responsibilities:
 
-Patch contributions can only be accepted through GitHub by creating a pull request from forked versions of our repositories. This allows us to review the contributions in a user friendly and reliable way, under public scrutiny.
+1. Check for CLA (Contributor License Agreement) compliance.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
 
-Please create separate pull requests for each concern; each pull request should have a clear unity of purpose. In particular, separate code formatting and style changes from functional changes. This makes each patch’s true contribution clearer and therefore quicker and easier to review.
+The current maintainers are:
 
-#### Reporting and fixing bugs
+* Anna Bridge [adbridge](https://os.mbed.com/users/AnnaBridge).
+* Martin Kojtal [0xc0170](https://os.mbed.com/users/Kojto).
+* Jimmy Brisson [theotherjimmy](https://os.mbed.com/users/theotherjimmy).
+* Shrikant Tudavekar [studavekar](https://os.mbed.com/users/shrikant1213).
+* Sam Grove [sg-](https://os.mbed.com/users/sam_grove).
+* Cruz Monrreal [cmonr](https://os.mbed.com/users/MrCruz).
 
-Before submitting a bug report or a bug fix, please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on it already.
+##### Stakeholders
```

Comment:
Query: Is this a new official term for lead engineers for new features and modules? I think it's vague because there are many stakeholders who do not fit this description.

Reply:
This was the name I could come up with. The groups are as follows:

Gatekeepers:
- maintainers
- tech leads/feature leads (often they match but not always)

I think the second group could be tech leads group, better?